Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -444,9 +444,16 @@ public let CodeGenerators: [CodeGenerator] = [
         b.beginTry() {
             b.generateRecursive()
         }
-        b.beginCatch() { _ in
-            b.generateRecursive()
-        }
+        withProbability(0.5, do: {
```

Comment:
I think this can now end up generating just 
```
try {
   ...;
}
```
which then wouldn't be valid JavaScript ("At least one catch-block, or a finally-block, must be present", from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#description). So you'll have to ensure that at least one of the two is generated. I think ideally, you'd generate try-catch, try-finally, and try-catch-finally statements with equal probability. WDYT?

Reply:
My bad! I've fixed that now with the latest commit. The math on the probability distribution may be a bit off, let me know if this needs tweaking :)

Reply:
Yeah I think the math should be correct. Alternatively, I think you could also write it as
```
b.beginTry() { b.generateRecursive() }
withEqualProbability({
    // try-catch-finally
    b.beginCatch() { _ b.generateRecursive() }
    b.beginFinally() { b.generateRecursive() }
}, {
    // try-catch
    b.beginCatch() { _ b.generateRecursive() }
}, {
    // try-finally
    b.beginFinally() { b.generateRecursive() }
})
```
Which might make the final probabilities a bit more obvious/intuitive (at the slight cost of duplicating a few lines).

Reply:
good shout! definitely more intuitive and easier to maintain.